app/Mcp/Tools/ListServices.php

```
@@ -30,35 +33,122 @@ public function handle(Request $request): Response
+                ->filter(fn (Service $service) => str_contains(strtolower((string) $service->status), strtolower(trim($status))))
+                ->values();
+            $total = $services->count();
+            $services = $services->slice($args['offset'], $args['per_page']);
+        } else {
+            $total = (clone $query)->count();
+            $services = $query
+                ->orderBy('name')
+                ->skip($args['offset'])
+                ->take($args['per_page'])
+                ->get();
+        }
```

PHPMD (2.15.0)

[error] The method handle uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)